RSP_Requirements_Design_Document/index.html

```diff
@@ -472,9 +505,23 @@
            windows inside the query body. We will analyze the window definitions later, but we already show how streams can be declared
            in the <code>FROM</code> clause, and thanks to a window identifier, they can be used in the query body:           
         </p>
-        <pre class="example highlight" title="RSP-QL: Declaration of the <code>ex:social</code> stream."><code>
+        <pre class="example highlight" title="RSP-QL: Declaration of a window <code>:win</code> for the <code>ex:social</code> stream."><code>
 FROM NAMED WINDOW :win ON ex:social [… window spec …] 
         </code></pre>
+
+        <p class="note">This example does not cover interesting cases where multiple streams, multiple windows on same stream, or combination
+          with named graphs are shown. <a href="https://github.com/streamreasoning/RSP-QL/issues/46">Issue</a></p>
+
+        <p>Notice that in a single query one may issue different windows for the same stream (e.g. with different window parameters,
+           such as size or slide), on multiple streams, or combined with RDF graphs. The following example shows a declaration of multiple
+           windows on several streams, combined with standard named graph declarations. We omit the window specifications, as they will be
+           covered later on.</p>
+       <pre class="example highlight" title="RSP-QL: Declaration of multiple windows for multiple streams."><code>
+FROM NAMED WINDOW :win1 ON ex:social  [… window spec …] 
+FROM NAMED WINDOW :win2 ON ex:social  [… window spec …] 
+FROM NAMED WINDOW :win3 ON ex:sensors [… window spec …] 
```

Comment:
given that is an example, would be worth to make some fake values to highlight the fact that :win1 and :win2 are different windows over the same stream